README.md

```diff
@@ -425,6 +425,15 @@ Individual test files can be run using the following command:
 npm run mocha -- path/to/test.js
 ```
 
+#### v1.6.0 Update
+
+Starting from version `1.6.0`, Lisk will be using [Jest](https://jestjs.io) as its main test runner and gradually deprecate [mocha](https://mochajs.org). Since, rewriting all existing mocha tests is not feasible at the moment, we are going to have 2 test runners in our code base until we refactor modules and move them to lisk-elements repository.
```

Comment:
In order to be the main test runner, Jest must cover most of the test. We shouldn't state this at this point because it's simply not true.

Reply:
> Lisk **will be** using

That's why I stated that from now on it will be the main test runner. Having legacy tests in mocha doesn't conflict with this.

Reply:
> until we refactor modules and move them to lisk-elements repository

I don't think that's true. Lisk Elements is libraries collection not the integration of libraries. Unless we are 100% sure about the direction, we should not mention such in readme. Below statement is good enough for now: 

> Since, rewriting all existing mocha tests is not feasible at the moment, we are going to have 2 test runners in our code base. Later we plan for further steps.